src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -2173,19 +2191,21 @@ private void DecodeWellKnownAttribute(ref DecodeWellKnownAttributeArguments<Attr
             }
             else if (attribute.IsTargetAttribute(this, AttributeDescription.AssemblyKeyFileAttribute))
             {
-                arguments.GetOrCreateData<CommonAssemblyWellKnownAttributeData>().AssemblyKeyFileAttributeSetting = (string)attribute.CommonConstructorArguments[0].ValueInternal;
+                arguments.GetOrCreateData<CommonAssemblyWellKnownAttributeData>().AssemblyKeyFileAttributeSetting = (string?)attribute.CommonConstructorArguments[0].ValueInternal;
             }
             else if (attribute.IsTargetAttribute(this, AttributeDescription.AssemblyKeyNameAttribute))
             {
-                arguments.GetOrCreateData<CommonAssemblyWellKnownAttributeData>().AssemblyKeyContainerAttributeSetting = (string)attribute.CommonConstructorArguments[0].ValueInternal;
+                arguments.GetOrCreateData<CommonAssemblyWellKnownAttributeData>().AssemblyKeyContainerAttributeSetting = (string?)attribute.CommonConstructorArguments[0].ValueInternal;
             }
             else if (attribute.IsTargetAttribute(this, AttributeDescription.AssemblyDelaySignAttribute))
             {
+#nullable disable // Can 'attribute.CommonConstructorArguments[0].Value' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
Same answer about standard libraries.

Reply:
I'm planning to leave this for a follow-up